spec/calculators/assessmentResultCalculatorSpec.js

```diff
@@ -1,18 +1,8 @@
-var Jed = require( "jed" );
 var AssessmentResultCalculator = require( "../../js/calculators/assessmentResultCalculator.js" );
 var MissingArgument = require( "../../js/errors/missingArgument.js" );
```

Comment:
This file should not be used anymore. Remove references after https://github.com/Yoast/YoastSEO.js/pull/321 has been merged.